sw/host/opentitanlib/src/backend/mod.rs

```diff
@@ -49,7 +53,8 @@ pub enum Error {
 /// Creates the requested backend interface according to [`BackendOpts`].
 pub fn create(args: &BackendOpts) -> Result<TransportWrapper> {
     let mut env = TransportWrapper::new(match args.interface.as_str() {
-        "" => create_empty_transport(),
+        "proxy" | "" => proxy::create(&args.proxy_opts),
```

Comment:
One last item (as discussed offline), since this is re-used by the session process, we probably want to have a different default for opentitantool vs opentitansession, since the session should connect to a real / emulated OT device, rather than another session.

Reply:
Done